docs/extensibility/actions.md

```diff
@@ -321,6 +321,14 @@ function my_function_callback( $order ) {
 
 add_action( 'woocommerce_blocks_checkout_order_processed', 'my_function_callback', 10 );
 ```
+<!-- FEEDBACK -->
+---
+
+[We're hiring!](https://woocommerce.com/careers/) Come work with us!
+
+🐞 Found a mistake, or have a suggestion? [Leave feedback about this document here.](https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/new?assignees=&labels=type%3A+documentation&template=--doc-feedback.md&title=Feedback%20on%20./docs/examples/checkout-order-processed.md)
+<!-- /FEEDBACK -->
+
```

Comment:
This section should be at the end of the file.

Reply:
Hey, @Aljullu !

Thanks for pointing that out!

It looks like `postbuild:docs` was failing for me because the function [syntax](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/c631ea42feb01f7598540ba68758c7086ff5350e/bin/add-doc-footer.sh#L2) in bin/add-doc.footer.sh requires bash as the runtime.

Even though the script starts with the `#!/bin/bash` shebang, this is irrelevant because the command in [package.json](https://github.com/AndrewJDawes/woocommerce-gutenberg-products-block/commit/ac6c7a2a56a5a780ec69e82fdc9ae7e66cda7171#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519) runs the script with `sh`, which may vary between OS (I'm on Ubuntu): `"postbuild:docs": "sh ./bin/add-doc-footer.sh"`.

Since /bin/sh on Ubuntu is symlinked to dash, not bash, running this command results in a syntax error since the `function` keyword is not defined in dash.

Since we need the script to run in bash and since we already have the shebang defined at the top, it seems the easiest way to make this consistent across platforms would be to change the postbuild:docs command to run the add-doc-footer.sh script directly instead of trying to run it with whatever `sh` is.

New commit below!


Please let me know if you have any questions or concerns!

Thanks!

Reply:
That makes sense, thanks for investigating this @AndrewJDawes!